lix/source-code/fs/src/memoryFs.ts

```diff
@@ -449,6 +512,10 @@ export function createNodeishMemoryFs(): NodeishFilesystem {
 				throw new FilesystemError("EISDIR", path, "unlink")
 			}
 
+			if ("placeholder" in parentDir || "placeholder" in target) {
+				throw new FilesystemError("EPLACEHOLDER", path, "readFile")
```

Comment:
I would throw "ENOTDIR" if the parent is a placeholder - if the target is a placholder we can't we drop it like we do in rm?

Reply:
really good catch :) good man